app/components/article-content.js

```diff
@@ -388,6 +391,51 @@ export default Component.extend(
 			}
 		},
 
+		/**
+		 * Opens a parent section of passed element if it's closed
+		 *
+		 * @returns {void}
+		 */
+		openSection(element) {
+			if (element) {
+				const section = element.closest('section[id*="section"]');
+				if (section) {
+					const header = section.previousElementSibling;
+					if (header) {
+						header.classList.add('open-section');
+					}
+				}
+			}
+		},
+
+		/**
+		 * Handles opening sections when click event occurs on CiteNote
+		 *
+		 * @returns {void}
+		 */
+		handleCiteNotes({target}) {
+			if (target.nodeName === 'A') {
+				if (target.hash.search('#cite_ref-') === 0) {
```

Comment:
How about combining these two if's into one? This way we could gain some readability as there would be less indentations:

```
if (target.nodeName === 'A' && target.hash.search('#cite_ref-') === 0) {
```